src/Components/Workspace/Transcripts/index.js

```diff
@@ -23,6 +23,19 @@ const Transcripts = ({ projectId, firebase }) => {
     return `/projects/${ projectId }/transcripts/${ id }/correct`;
   };
 
+  const usePreventWindowUnload = (uploading) => {
+    useEffect(() => {
+      if (!uploading) return;
+      window.onbeforeunload = () => {
+        return '';
+      };
+    }, [ uploading ]);
+  };
+  const CheckIfUploading = () => {
```

Comment:
I think you only want to capitalise consts that are components in React :)

```suggestion
  const checkIfUploading = () => {
```